doc/versionHistory.rst

```diff
@@ -6,6 +6,16 @@
 Version History
 ##################
 
+.. _lsst.ts.ofc-3.0.1:
+
+v3.0.1
+======
+
+* Removed Zemax Coordinate System, which is substituted by the Camera Coordinate System. 
+* Updates the bending modes used to the new uncorrupted bending modes.
+* Updates sensitivity matrix and intrinsic zernikes for the new coordinate system and new bending modes.
+* Removes old unused test data.
```

Comment:
Please help to use the past tense to align with others.

Updates --> Updated
Removes --> Removed